analyzer/windows/modules/packages/exe.py

```diff
@@ -11,6 +11,28 @@
 class Exe(Package):
     """EXE analysis package."""
 
+    REGKEYS = [
+        [
+            HKEY_LOCAL_MACHINE,
+            "Software\\Microsoft\\Security Center",
+            {
+                # "Would you like Internet Explorer as default browser?"
```

Comment:
that's one option however then you have to manage your template(s) also ;) anyways it doesn't really matter - it's mostly about about the taste how people like to manage their infra. As vmcloak is also quite dynamic then my wild guess is that "profiles" could be the optimal sweetspot as they can be reused on both.